### PR TITLE
Updated translation

### PR DESCRIPTION
**Pull request by guidomazzone**
Updated to latest template.pot and improved translation.
In order to give the maximum quality to the translation I asked you two questions here:
https://github.com/Zren/plasma-applet-eventcalendar/commit/b8d7d6562508e6e56b6308331bb04fa9a83ae8e7#commitcomment-43969486

### CODE REVIEW

File: package/translate/it.po [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-# Guido Mazzone <guido.mazzone93@outlook.it>, 2020.
+# FIRST AUTHOR Guido Mazzone <guido.mazzone93@outlook.it>, 2020.
 #
 msgid ""
 msgstr ""
 "Project-Id-Version: eventcalendar\n"
 "Report-Msgid-Bugs-To: https://github.com/Zren/plasma-applet-eventcalendar\n"
 "POT-Creation-Date: 2020-11-07 13:45-0500\n"
-"PO-Revision-Date: 2020-11-02 13:30+ZONE\n"
+"PO-Revision-Date: 2020-11-07 21:35 (GMT+1)\n"
```
Comment (Zren) [inline package/translate/it.po:11]: `+0100`

File: package/translate/it.po [hunk 2]
```diff
@@ -1,16 +1,16 @@
-# Translation of eventcalendar in it
+# Translation of eventcalendar in Italian
 # Copyright (C) 2020
 # This file is distributed under the same license as the eventcalendar package.
-# Guido Mazzone <guido.mazzone93@outlook.it>, 2020.
+# FIRST AUTHOR Guido Mazzone <guido.mazzone93@outlook.it>, 2020.
```
Comment (Zren) [inline package/translate/it.po:4]: `FIRST AUTHOR` isn't needed here since you put your name

**Review by Zren — COMMENTED** [review]
I can fix these after merging.

**Comment by guidomazzone** [comment]
ok thanks for your fixes

**Pull request merged**